08-components-lab/src/02-numberguessing.jsx

```diff
@@ -1,17 +1,53 @@
-/**
- * Lab 02:
- *
- * Implement a number guessing game component.
- *
- * The component should provide an input field for the user to guess a
- * number picked randomly by the computer (from 1 to 1,000).
- * When typing a number, the component should print "too low" or "too high" 
- * as a hint to the user.
- *
- * To make the game interesting, computer should sometimes cheat and print wrong messages.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+import _ from 'underscore';
+
+export default class NumberGuessing extends React.Component {
+    randomNum = _.random(1, 1000);
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: 0,
+            output: "",
+        };
+    }
+
+    myXOR(a,b) {
+        return ( a || b ) && !( a && b );
+    }
+
+    handleChange(e) {
+        const value = e.target.value;

```

Comment:
You can use ES6 destructuring like this:
`const { value } = e.target;`
or use ES6 function arguments destructuring in the following manner:

`handleChange({ target: { value } })`